backend/src/main/java/com/sim_backend/websockets/observer/AuthorizeObserver.java

```diff
@@ -0,0 +1,28 @@
+package com.sim_backend.websockets.observer;
+
+import com.sim_backend.websockets.events.*;
+import com.sim_backend.websockets.exceptions.*;
+import com.sim_backend.websockets.messages.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class AuthorizeObserver {
+  private List<OCPPMessageHandler> observers = new ArrayList<>();
```

Comment:
Observers should not be tracked/registered separately here. The WebSocket client has a function that allows for registering observers already (see `OCPPWebSocketClient.java`). Just register this observer in our main.